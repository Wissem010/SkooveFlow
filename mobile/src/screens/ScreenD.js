//react imports
import React from 'react';
//Ui imports
import {View, StyleSheet, Image} from 'react-native';

export default function ScreenD() {
  return (
    <View style={styles.backgroundStyle}>
      <Image height={64} source={require('../assets/check.png')} />
    </View>
  );
}

const styles = StyleSheet.create({
  backgroundStyle: {
    backgroundColor: 'lightblue',
    height: '100%',
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
  },
});
