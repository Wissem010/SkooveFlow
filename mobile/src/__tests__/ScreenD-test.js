import React from 'react';
import renderer from 'react-test-renderer';
import ScreenD from '../screens/ScreenD';

test('renders correctly', () => {
  const tree = renderer.create(<ScreenD />).toJSON();
  expect(tree).toMatchSnapshot();
});
