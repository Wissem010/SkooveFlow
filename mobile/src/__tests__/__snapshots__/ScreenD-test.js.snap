// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "lightblue",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <Image
    height={64}
    source={
      Object {
        "testUri": "../../../src/assets/check.png",
      }
    }
  />
</View>
`;
